src/frontend/src/pages/Layout.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import AppHeader from "../components/AppHeader/AppHeader";
+import { Content } from "carbon-components-react";
+import carbon__themes from "@carbon/themes";
+
+interface Props {
```

Comment:
I am not too familiar with react. However for that interface it will only be used in Layout.tsx. So I dont feel like there is a need to import them as a type from somewhere as the prop interface may vary depending on the component.